.buildkite/pipeline.yml

```diff
@@ -115,7 +115,7 @@ steps:
 
       # Upload the built artifacts.
       - cd /var/tmp/artifacts/x86_64-fortanix-unknown-sgx/debug
-      - buildkite-agent artifact upload oasis-runtime-keymanager.sgxs
+      - buildkite-agent artifact upload oasis-ethwasi-runtime-keymanager.sgxs
     env:
       RUNTIME_VARIANT: sgxs
     agents:
```

Comment:
you can also rename the trigger bellow (oasis-runtime-build-image-context). (and then before merging this PR also the buildkite pipeline name (through buildkite ui), otherwise the pipeline upload will fail.

Reply:
Ah thanks, will do